ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+daemon off;
+
+error_log  /dev/stdout warn;
+pid        {{ .WorkingDir }}/nginx.pid;
+
+events {
+ worker_connections  1024;
```

Comment:
We should also use epoll and enable multi_accept.

See https://www.linode.com/docs/websites/nginx/configure-nginx-for-optimized-performance